Tests/StreamChatTests/APIClient/Endpoints/Payloads/IdentifiableModel_Tests.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class IdentifiableModel_Tests: XCTestCase {
+    var database: DatabaseContainer!
+
+    private lazy var deviceDTO: DeviceDTO = {
```

Comment:
We should use the existing `TestManagedObject` as the DTO for unit tests when the test itself is irrelevant of a specific DTO

Reply:
It is throwing an error when using TestManagedObject because it cannot get its entity description

`An NSManagedObject of class 'TestManagedObject' must have a valid NSEntityDescription`

Reply:
Alright, then we should have another DTO for this test. Basically, it shouldn't be bound to any DTO config